src/Foundation/ContainerUtil.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Foundation;
+
+class ContainerUtil
+{
+    public static function wrapCallback($callback, $container)
```

Comment:
I think we need a comment here explaining the use case for this function.

Something like "Wraps a callback so that string-based invokable classes get resolved only when actually used".

Also, do we want an `@internal` phpdoc to discourage extensions from calling this directly?